resource-definitions/terraform-driver/runner/s3-ext-runner-refs.tf

```diff
@@ -0,0 +1,72 @@
+resource "humanitec_resource_definition" "aws_terraform_external_runner_resource_s3_bucket" {
+  id          = "aws-terrafom-ext-runner-s3-bucket"
+  name        = "aws-terrafom-ext-runner-s3-bucket"
+  type        = "s3"
+  driver_type = "humanitec/terraform"
+  # The driver_account references a Cloud Account configured in the Platform Orchestrator.
+  # Replace with the name of your AWS Cloud Account.
+  driver_account = "my-aws-account"
```

Comment:
Why do we need the `driver_account`? When using the runner, it's getting its cluster credentials from referencing the `k8s-cluster.runner` cluster.

Reply:
We need it for the backend configuration